Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [workspace]
-members = ["antlr", "interpreter", "example", "fuzz"]
+members = ["cel", "example", "fuzz"]
```

Comment:
Should we unify everything at this level? More as single/simple [cargo project layout](https://doc.rust-lang.org/cargo/guide/project-layout.html) ?
`cel` would be `src`
`example` -> `examples`
`fuzz` would go under `tests`

Reply:
Ah, yeah... well, actually you can keep every example under `examples/` in their own dir & have these have their own `Cargo.toml` and as such their own deps. Now I don't know if `cargo --examples` & `cargo --example axum` work in those cases tho, but cargo does work from within `example/axum` tho. I can investigate that and let you know...